features/levering-koopsom.feature

```diff
@@ -0,0 +1,152 @@
+Functionaliteit: levering koopsom
+
+    Achtergrond:
+        Gegeven de registratie BRK kent de onroerendezaak met kadastraalonroerendezaakidentificatie: NL.IMKAD.KadastraalObject.76870487970000,
+        En waarvan de volgende velden gevuld zijn: identificatie, type, kadastraleaanduiding, koopsom, koopjaar, aardCultuurbebouwd
+        En de onroerendezaak met kadastraalonroerendezaakidentificatie: NL.IMKAD.KadastraalObject.76870488070000,
+        En waarvan de volgende velden gevuld zijn: identificatie, type, kadastraleaanduiding, koopsom, aardCultuurbebouwd
+
+    Scenario: Onroerendezaak (met koopsom & koopjaar & aardCultuurbebouwd) wordt opgevraagd zonder fields parameter
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000
+        Dan worden de koopsom & koopjaar & aardCultuurBebouwd teruggegeven
+        En wordt de volgende json response teruggegeven:
+        """
+        {
+            "identificatie": "NL.IMKAD.KadastraalObject.76870487970000",
+            "koopsom": {
+                "koopsom": 185000,
+                "koopjaar": 2015
+            },
+            "type": "perceel",
+            "aardCultuurBebouwd": {
+                "code": "11",
+                "waarde": "Wonen"
+            },
+            "kadastraleAanduiding": {
+                "kadastraleGemeente": {
+                    "code": "87",
+                    "waarde": "Beekbergen"
+                },
+                "perceelnummer": 4879,
+                "sectie": "K"
+            },
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000"
+                }
+            }
+        }
+        """
+
+
+    Scenario: Onroerendezaak (met koopsom & koopjaar & aardCultuurbebouwd) wordt opgevraagd met fields parameter met 1 van die drie velden erin
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000?fields=koopsom
```

Comment:
ook een interessant scenario is met fields=aardCultuurOnbebouwd

worden dan ook koopsom & koopjaar & aardCultuurBebouwd teruggegeven?

Reply:
Je kan van deze scenario een abstract scenario kunnen maken. Daarmee kan je laten zien dat je bij fields=koopsom, fields=koopjaar of fields=aardCultuurbebouwd altijd koopsom, koopjaar en aardCultuurBebouwd samen krijgt